docs/source/configuration/environmentvariables.md

```diff
@@ -89,3 +89,21 @@ It displays the errors of the non-compliant customizations (in server console) i
 #### Internationalization errors (i18n)
 
 It will enable the log of missing i18n messages (in console).
+
+### Add addons via environment variable
+
+Quite often, you need different configurations and enabling components (eg. for testing purposes). You can use the `ADDONS` environment variable to define them. They sum up to the ones set in `package.json` or programatically in `volto.config.js`. You can specify released (published) packages (installed previously in your environment, and present already in node_modules), or "packages" folder local to your project (not in development, but optional, eg. Volto's testing packages).
```

Comment:
```suggestion
Quite often, you need different configurations and enabling components (for example, testing purposes).
You can use the `ADDONS` environment variable to define them.
They sum up to the ones set in `package.json` or programatically in `volto.config.js`.
You can specify released (published) packages (installed previously in your environment, and present already in `node_modules`), or `packages` folder local to your project (not in development, but optional, for example, Volto's testing packages).
```

This last sentence is difficult to comprehend with too many parenthetical statements. Would you please simplify and clarify it?

Reply:
Will commit the suggestion then think about it a bit more.